clang/tools/dpct/DpctOptRules/cmake_script_migration_rule.yaml

```diff
@@ -41,3 +41,51 @@
       In: ${arg}.cu
       Out: ${arg}.dp.cpp
 
+- Rule: rule_cmake_cxx_standard
+  Kind: CMakeRule
+  Priority: Fallback
+  CmakeSyntax: cmake_cxx_standard
+  In: set (CMAKE_CXX_STANDARD ${version})
+  Out: set (CMAKE_CXX_STANDARD 17)
+
+- Rule: rule_cmake_cxx_compiler
+  Kind: CMakeRule
+  Priority: Fallback
+  CmakeSyntax: cmake_cxx_compiler
+  In: set (CMAKE_CXX_COMPILER ${version})
+  Out: set (CMAKE_CXX_COMPILER icpx)
+
+- Rule: rule_set_property_cxx_standard
+  Kind: CMakeRule
+  Priority: Fallback
+  CmakeSyntax: set_property_cxx_standard
+  In: set_target_properties(${targets} PROPERTIES CXX_STANDARD ${version})
+  Out: set_target_properties(${targets} PROPERTIES CXX_STANDARD 17)
+
+- Rule: rule_set_property_cuda_seprarable_only
+  Kind: CMakeRule
+  Priority: Fallback
+  CmakeSyntax: set_property_cuda_seprarable_only
+  In: set_target_properties(${target_property} CUDA_SEPARABLE_COMPILATION ${state})
+  Out: ""
```

Comment:
For line 70: suggest to use 
```
Out: |
    #[[
      set_target_properties(${target_property} CUDA_SEPARABLE_COMPILATION ${state})
    ]]
```
which will comment the cuda syntax in the migrated script, which is more user friend than  being removed.

Reply:
Commenting out the code is much better! Using the comment format as mentioned here

https://cmake.org/cmake/help/latest/manual/cmake-language.7.html#bracket-comment